src/ts/vttutils.ts

```diff
@@ -77,35 +94,55 @@ const setVttLine = (
     relativeLinePosition = (100 * absoluteLinePosition) / subtitleOverLaySize.height;
   }
 
-  cueContainerDom.css(direction, `${relativeLinePosition}%`);
-  setVttLineAlign(cueContainerDom, vtt, direction);
+  if (vtt.lineAlign !== 'end')
+    cueContainerDom.css(
+      overlayReferenceEdge, `${relativeLinePosition}%`);
```

Comment:
since the offset passes to `setVttLineAlign()` there is only need to set this for start alignment. Otherwise this css property is set 2 times for center alignment while 1 time is sufficient.